docs/source/configuration/overview.mdx

```diff
@@ -48,7 +48,7 @@ The graph ref for the GraphOS graph and variant that the router fetches its supe
 
 The [graph API key](/graphos/api-keys/#graph-api-keys) that the router should use to authenticate with GraphOS when fetching its supergraph schema.
 
-**Required** if using managed federation.
+If starting the router with an [offline license](#--license), setting both `APOLLO_GRAPH_REF` and `APOLLO_KEY` configures the router to send usage metrics to GraphOS on a best-effort basis; not setting both disables sending usage metrics. Otherwise, setting this is **required** when using managed federation.
```

Comment:
Since this applies to both and is repeated, it might make more sense to put it in a subsection below the table. Then, in each row in the table it can say something like this:

```suggestion
**Required** when using [managed federation](/federation/managed-federation/overview/) except when using an [offline license](#offline-license) to run the router.
```